src/content/en/updates/2018/09/asynchronous-access-to-http-cookies.md

```diff
@@ -146,11 +146,11 @@ The Cookie Store API brings an alternative method for observing cookie
 changes, which does not require polling.
 
     cookieStore.addEventListener('change', (event) => {
-      for (const cookie in event.changed) {
+      for (const cookie of event.changed) {
```

Comment:
@surma - can you comment on the `of`/`in` difference?

Reply:
If `changed` is an array, `in` would iteration over the _indices_, while`of` would iterator over the items in the array. Looking at the body, `of` seems indeed to be intended here.

Reply:
Indeed, `event.changed` and `event.deleted` are arrays. Using `for...in` would iterate over all the enumerable properties of the array, which in this case are just the indexes. While using `for...of` would iterate through all the values of the array, which is what was intended if we look at the body of the `for` loop.